File: FLTranslucentNavigationBar_OCDemo/FLTranslucentNavigationBar_OC/ViewController.m
//
//  ViewController.m
//  FLTranslucentNavigationBar_OC
//
//  Created by clarence on 16/9/2.
//  Copyright © 2016年 clarence. All rights reserved.
//

#import "ViewController.h"
#import "SecondViewController.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.title = @"first viewController";
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event{
    SecondViewController *vc = [[SecondViewController alloc] init];
    [self.navigationController pushViewController:vc animated:YES];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: FLTranslucentNavigationBar_swiftDemo/FLTranslucentNavigationBar_swift/ViewController.swift
//
//  ViewController.swift
//  FLTranslucentNavigationBar_swift
//
//  Created by clarence on 16/9/2.
//  Copyright © 2016年 clarence. All rights reserved.
//

import UIKit

class ViewController: FLTranlucentRootViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        title = "小咧咧"
    }

    override func touchesBegan(touches: Set<UITouch>, withEvent event: UIEvent?) {
        let vc = SecondViewController()
        navigationController?.pushViewController(vc, animated: true)
    }


}

